src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -698,6 +706,28 @@ public BitbucketTeam getTeam() throws IOException {
         }
     }
 
+    /**
+     * Get Team avatar
+     */
+    @Override
+    public AvatarImage getTeamAvatar() throws IOException {
+        if (userCentric) {
+            return null;
```

Comment:
Sure there is no way to get the avatar? :)

Reply:
TBH, I copied this and, in an effort to reduce the scope of my changes,  have not even thought about it. I assume "userCentric" means the "team" is actually user personal repos? I am sure there is a way to get avatar in that case - want me to try?

Reply:
nut unless it was already null 😆

Reply:
Yes, I do not believe it ever worked as that condition was lifted from Server's `getTeam()`code. I just looked into it, and it is a bit more involved to add:-(. The v1 API used by server is warped, and it a method for setting the user's avatar but not getting it. I did find a potential way of getting it that is more like Cloud, but it involved getting/parsing user info - which is not there at all. Its not hard to add, but will be a lot of extra code and will have to be done twice (I suspect the cloud version probably does not work either)

This brings me to another point - for what its worth, the plugin is a bit asymmetric between cloud (looks like the original) and server (added later?) implementations. It would be really nice to bring it into symmetry and create parallel package structure to start with and then move shared code into one place. That would make this a bit easier (no need to implement things twice, slightly differently) and make it much easier to have feature parity and consistency between the two implementations.  I thought about doing that, but moving packages around AND introducing changes at the same time seemed like a very bad idea.

Reply:
I agree but that's besides the scope of this PR 😅

Reply:
Agree. So, lets merge ;-) Also - any idea as to how do I get someone to take a look at the scm-api-plugin PR? Neither JIRA issue nor PR got any attention from anyone :-( Once that is merged, we would need to remove duplicate code from this plugin